web-ui/src/pages/PulseReportPage.jsx

```diff
@@ -244,10 +244,10 @@ const PulseReportPage = () => {
   };
 
   useEffect(() => {
-    //TODO: Skipping the permission check during testing
-    //      because this permission has not been implemented yet.
-    // if (selectHasPulseReportPermission(state)) loadPulses();
+    // TODO: Uncomment this check after PR #2429 is merged.
```

Comment:
I just merged https://github.com/objectcomputing/check-ins/pull/2429

Reply:
As far as I can tell, it's working 🎉 

Do we need a `PulseReportPage.test.jsx` though 🤔

Reply:
@timyates I added this test.

Reply:
Pushed a fix for the test here https://github.com/objectcomputing/check-ins/pull/2479 the date was changing in the snapshot when under test